### PR TITLE
Fix split=None in the train_test_split tool

### PR DESCRIPTION
**Pull request by anuprulez**
Fixes the issue: https://github.com/bgruening/galaxytools/issues/1801

Also, adds a test for `shuffle=None`

ping @gdefazio @bgruening thanks!

### CODE REVIEW

**Comment by anuprulez** [comment]
The list of tools (25 out of 35) that fail on `planemo lint`: 

clf_metrics.xml
discriminant.xml
ensemble.xml 
feature_selection.xml 
fitted_model_eval.xml
generalized_linear.xml
keras_batch_models.xml
keras_model_builder.xml
keras_train_and_eval.xml
lightgbm.xml
ml_visualization_ex.xml
model_prediction.xml
model_validation.xml
nn_classifier.xml
numeric_clustering.xml
pairwise_metrics.xml
pca.xml
pipeline.xml
regression_metrics.xml
sample_generator.xml
search_model_validation.xml
simple_model_fit.xml
sparse.xml
svm.xml
train_test_eval.xml
